pkg/plugin/testdata/plugdir/good/hello/plugin.yaml

```diff
@@ -3,7 +3,26 @@ version: "0.1.0"
 usage: "usage"
 description: |-
   description
-command: "$HELM_PLUGIN_DIR/hello.sh"
+platformCommand:
+  - os: linux
+    arch:
+    command: "sh"
+    args: ["-c", "${HELM_PLUGIN_DIR}/hello.sh"]
+  - os: windows
+    arch:
+    command: "pwsh"
+    args: ["-c", "${HELM_PLUGIN_DIR}/hello.ps1"]
+command: "${HELM_PLUGIN_DIR}/hello.sh"
 ignoreFlags: true
+platformHooks:
+  install:
+    - os: linux
+      arch:
```

Comment:
```suggestion
```

possible to elide the empty fields? (here and other places)

Reply:
Done this one, but I'm not sure of any other instances in the new code.